js/techtree.js

```diff
@@ -371,13 +371,14 @@ class Tree {
         this.padding = 10;
         this.element_height = 0;
         this.lanes = [];
+        this.offset_x = 150;  // 150 is starting offset from the left to accommodate age icons
     }
 
     updateOffsets() {
         this.element_height = this.height / 4 / 3;
         let element_offset = this.element_height / 2;
 
-        this.offsets.dark_1 = this.padding;
+        this.offsets.dark_1 = this.padding + 10;  // shift the tech tree down by 10px to align center in the row
```

Comment:
I would instead increase `this.padding` to 20. Yes, that also increases the distance between vertical building lanes, but it is still not as big as the one in the original game.